main.py

```diff
@@ -56,7 +56,7 @@ def main(java_driver_git, scylla_install_dir, tests, versions, driver_type,scyll
 def extract_n_latest_repo_tags(repo_directory: str, major_versions: List[str], latest_tags_size: int = 2,
                                is_scylla_driver: bool = True) -> List[str]:
     major_versions = sorted(major_versions, key=lambda major_ver: float(major_ver))
-    filter_version = f"| grep {'' if is_scylla_driver else '-v '}'.*\..*\..*\.'"
+    filter_version = f"| grep {'' if is_scylla_driver else '-v '}'.*\..*\..*\.' | grep -v '[-_x]'"
```

Comment:
`-v` for scylla does not make any sense, let's fix it
```suggestion
    filter_version = "| grep -P '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+'" if is_scylla_driver else "| grep -P '[0-9]+\.[0-9]+\.[0-9]+'"
```

Reply:
This suggestion will match for example datastax drivers with scylla versions because if they have 4 sections, then they have at least 3 sections too.

Reply:
This is valid concern, but `scylla-pkg` runs datastax tests from datastax repo, so it won't break anything.
To address it you can wrap regexp into '^<regexp>$':
```suggestion
    filter_version = "| grep -P '^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$'" if is_scylla_driver else "| grep -P '^[0-9]+\.[0-9]+\.[0-9]+$'"
```

Reply:
@dkropachev @Bouncheck feel free to take over the PR, and change.

I just noticed those weird versions, and it was annoying me